var/spack/repos/builtin/packages/py-scine-chemoton/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyScineChemoton(PythonPackage):
+    """Software driving the automated exploration of chemical reaction networks"""
+
+    homepage = "https://scine.ethz.ch/download/chemoton"
+    pypi = "scine_chemoton/scine_chemoton-2.2.0.tar.gz"
+
+    version("2.2.0", "cda4f9de1e1c00ecc5e0b2d9c17a5edb56b468d454022e3f4045ec116ba2ec45")
+
+    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-numpy", type="run")
+    depends_on("py-scine-database", type="run")
+    depends_on("py-scine-molassembler", type="run")
+    depends_on("py-scine-utilities", type="run")
+    depends_on("py-scipy", type="run")
+    depends_on("py-setproctitle", type="run")
```

Comment:
These should all be `type=("build", "run")`. Some build tools check to make sure the dependencies exist in `PYTHONPATH` before building. Also, our import tests try to import all files during install and will require these to be available.